ext/fetch/23_request.js

```diff
@@ -353,7 +354,9 @@
         request = cloneInnerRequest(originalReq, true);
         request.redirectCount = 0; // reset to 0 - cloneInnerRequest copies the value
         signal = input[_signal];
+        isRelative = originalReq.isRelative;
       }
+      ObjectDefineProperty(request, "isRelative", {value: isRelative, writable: false, enumerable: false, configurable: false});
```

Comment:
A comment would be nice here to clarify why we define it like this rather than just doing `request.isRelative = isRelative`